docs/manage/remotes.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Parts, Sub-Parts and Remotes"
+linkTitle: "Remotes and Sub-Parts"
+weight: 40
+type: "docs"
+description: "Connect robots to each other."
+tags: ["server", "components", "services"]
+---
+
+When robots can communicate with each other, they can share resources and operate collaboratively.
+This document explains how to establish secure connections between robots.
+
+### Robot parts
+
+Robots are organized into *parts*, where each part represents a computer (a single-board computer like a Raspberry Pi or a desktop, laptop, or other computer) running `viam-server`, the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.

```

Comment:
```suggestion
Robots are organized into *parts*, where each part represents a computer (a [single-board computer](/installation/prepare) or a desktop, laptop, or other computer) running `viam-server`, the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.

```
We want to stop calling out Raspberry Pi specifically as our aim is to support many SBCs

Reply:
Done on the other PR